soh/soh/Enhancements/randomizer/settings.cpp

```diff
@@ -875,7 +874,6 @@ void Settings::CreateOptions() {
     mOptionGroups[RSG_TIMESAVERS_IMGUI] = OptionGroup::SubGroup("Timesavers", {
         &mOptions[RSK_CUCCO_COUNT],
         &mOptions[RSK_BIG_POE_COUNT],
-        &mOptions[RSK_SKIP_CHILD_STEALTH],
         &mOptions[RSK_SKIP_CHILD_ZELDA],
```

Comment:
Could we get both of the skip child stealth RSK entries commented out instead with an explanation that it's not currently needed but that it's kept around until freestanding needs it?